mappings/net/minecraft/entity/ai/brain/Activity.mapping

```diff
@@ -3,6 +3,8 @@ CLASS net/minecraft/class_4168 net/minecraft/entity/ai/brain/Activity
 	FIELD field_23827 hashCode I
 	METHOD <init> (Ljava/lang/String;)V
 		ARG 1 id
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
```

Comment:
This isnt really needed, or use `other`

Reply:
`o` is the name from `java.lang.Object`'s source code. I support sollace.

Reply:
It's okay to use `o` in this PR. We need a separate PR for refactoring this thing anyway (#2299).